src/games/RomUtils.hpp

```diff
@@ -22,6 +22,10 @@ class System;
 // reads a byte at a memory location between 0 and 1023
 extern int readRam(const System* system, int offset);
 
+// writes a byte at a memory location between 0 and 128,
+// this method is required to allow one to set a game mode
+extern void writeRam(System* system, int offset, unsigned char value);
```

Comment:
I thought modes were set by repeatedly pressing select? Are you thinking it's easier to directly write to RAM?

Reply:
I was following the previous pull requests and this was in there. If I remember correctly, we used this initially for debugging and it was left in there. I'll remove it for now. I don't think I'll need it.